draft-ietf-tsvwg-multipath-dccp.mkd

```diff
@@ -724,10 +724,13 @@ The MP_RTT option is used to transmit RTT values in milliseconds and
 MUST belong to the path over which this information is transmitted.
 Additionally, the age of the measurement is specified in milliseconds.
 
+The RTT and Age information is a 32-bit integer that increases monotonically with
+time, at a rate of 1 unit per 1 millisecond.  At this rate,
+RTT/Age Value will cover approximately 1193 hours.
+
```

Comment:
This reads strangely to me. What are we trying to say here? That the Age and RTT are represented as 32-bit integers? And that this allows to capture a maximum value for the RTT and Age of approximately 1193 hours. We already say above that they use milliseconds as the unit, so I do not understand the part about the rate.

Reply:
Please look into https://github.com/markusa/ietf-multipath-dccp/pull/52/commits/fca488a5d37a2e5d3e67745b5a136b608fb73998